src/partials/how-we-do.html

```diff
@@ -0,0 +1,118 @@
+    <section class="how-we-do__section">
+      <div class="container">
+        <h1 class="how-we-do__heading">How do we do it</h1>
+        <ul class="how-we-do__list">
+          <li class="how-we-do__item">
+            <div class="how-we-do__svg-list">
+              <svg class="how-we-do__svg">
+                <use href="/images/sprite.svg#icon-study-program"></use>
+              </svg>
+            </div>
+            <ul class="how-we-do__text">
+              <li class="how-we-do__list-text">
+                <h3 class="how-we-do__list-heading">Adapted program preparation for UPE</h3>
```

Comment:
Klasa w tym h3 i następnych dwóch p moim zdaniem powinna nazywać się "how-we-do__item-" zamiast "how-we-do__list-". Nagłówek oraz paragrafy są wewnątrz itemu, a nie ogólne dla całej listy, how-we-do__list-heading w moim odczuciu brzmi, jakby cała lista miała jeden główny nagłówek, który ta klasa styluje.